docs/editing_guidelines.md

```diff
@@ -0,0 +1,138 @@
+# Guidelines for CL editors
+
+## Naming terms
+
+### General rules
+1. All names should be singular nouns.
+2. Names should not be capitalised, unless they are or contain a proper name (e.g. _Johnston's organ_).
```

Comment:
```suggestion
2. Names should not be capitalised unless they are or contain a proper name (e.g., _Johnston's organ_).
```

Reply:
This is explicitly a rule about naming.  The change makes is a rule about words.

Reply:
I guess the context helps - but the main aim is to ensure editors avoid the habit of capitalising the first letter of a full name.